assets/js/hocs/with-product.js

```diff
@@ -36,10 +36,15 @@ const withProduct = createHigherOrderComponent( ( OriginalComponent ) => {
 		}
 
 		loadProduct() {
-			const { productId } = this.props.attributes;
+			const { productId, previewProduct } = this.props.attributes;
```

Comment:
Is there any benefit on saving `previewProduct` in the `attributes` instead of importing it directly from a JSON file?

Reply:
I think it's a good idea to have a js file containing common fallback preview info for things like products/reviews etc.  We can then have a common set of previews to keep things consistent everywhere.  We could start with product (including the one Mike did for his product block)? So a js with something like:

```js
export const productPreview = {
  name: 'My Awesome Product',
  // other properties matching the product object shape returned from rest
  // endpoint
};
```

Maybe located in `assets/js/previews`?

However, the upside of making this an attribute is the possibility for previews to be swapped out by extensions if they wanted... so something to consider there.  We could default to the json on the attribute (for consistency) but future iterations we could make this extensible so extensions can swap the preview for something else?

Reply:
I agree that it would be better to display a preview instead of a message, if it doesn't require too much additional work to implement and maintain the data required for a fallback preview of any block.